docs/en/ingest-management/release-notes/release-notes-8.2.asciidoc

```diff
@@ -0,0 +1,248 @@
+`@mappings` component templates are not removed. These templates are deprecated
+in favor of the new `@package` component template. 
+
+*Impact* +
+
+The legacy templates do not cause issues, but you might want to deleted them
+because they are no longer used.
+
+To find and delete legacy component templates:
+
+. In {kib}, go to *Stack Management > Index Management* and click
+*Component Templates*.
+
+. Click `Not in use` to filter out component templates that are still in use.
+
+. If the list contains component templates that end in `@mappings` or
+`@settings`, use the Actions menu to delete them.
+
+====
+
+//REVIEWERS: Are there any //deprecations we need to document here? It sounds
+//like some component templates were deprecated, but not sure when.
+
+//keeping the template until I know whether we need it.
+
+//[discrete]
+//[[deprecations-8.2.0]]
+//=== Deprecations
+
+//The following functionality is deprecated in 8.2.0, and will be removed in
+//8.2.0. Deprecated functionality does not have an immediate impact on your
+//application, but we strongly recommend you make the necessary updates after you
+//upgrade to 8.2.0.
+
+//{fleet}::
+//* add info
+
+//{agent}::
+//* add info
+
+[discrete]
+[[enhancements-8.2.0]]
+=== Enhancements
+
+{fleet}::
+* beta[] Add a {ls} output type to {fleet} settings. {kib-issue}104987[#104987]
+* Merge the settings and mappings component templates into `@package`.
+{kib-pull}128498[#128498]
+* Redesign the agent flyout. {kib-pull}128381[#128381]
+* Add a link from agent details page to agent dashboard. {kib-pull}127882[#127882]
+* Update add agent instructions in {fleet}-managed mode for Kubernetes. {kib-pull}127703[#127703]
+* Add `time_series_metric` mapping for `metric_type` package field. {kib-pull}126322[#126322]
+* Add support for dimension field. {kib-pull}[#126257]
+* Refactor auto upgrade package policies logic. {kib-pull}125909[#125909]
+* Move mappings from index template to component template. {kib-pull}124013[#124013]
+
+{agent}::
+* Update ack response schema and processing; add retrier for acks. {agent-pull}200[#200]
+* Enhance error messages and logs for process start. {agent-pull}225[#225]
+* Change the default policy selection logic. When the agent has no policy ID or
+name defined, it will fall back to defaults (defined by
+`$FLEET_SERVER_POLICY_ID` and `$FLEET_DEFAULT_TOKEN_POLICY_NAME` environment
+variables respectively). {agent-pull}226[#226]
+* Add Elastic APM instrumentation. {agent-pull}180[#180]
+* Agent can be built for `darwin/arm64`. When it's built for both
+`darwin/arm64` and `darwin/adm64` a universal binary is also built and packaged. {agent-pull}203[#203]
+* Add support for Cloudbeat. {agent-pull}179[#179]
+* Fix download verification in snapshot builds. {agent-issue}252[#252]
+* Add support for kubernetes cronjobs. {agent-pull}279[#279]
+
+[discrete]
+[[bug-fixes-8.2.0]]
+=== Bug fixes
+
+{fleet}::
+* There are no bug fixes for {fleet} in this release.
```

Comment:
I am surprised that this is empty, but I see that all of our 8.2 fixes also target older patch releases (https://github.com/elastic/kibana/pulls?q=is%3Apr+label%3ATeam%3AFleet+label%3Av8.2.0+label%3Arelease_note%3Afix+is%3Aclosed), so maybe they are covered in release notes for the older versions?

Reply:
Maybe. TBH, I just take the release notes after they are created for Kibana and copy them over to our docs (so users can find them in one place). I think their process is all script-driven, but I can make sure the bug fixes are represented in an earlier version if you'd like.

Reply:
It would be great to verify the process on Kibana's scripting side, thanks!

Reply:
@jen-huang I went through the older release notes and confirmed that all the 8.2 issues are marked as fixed in earlier releases.  I can see how this might be potentially confusing to users, but this is expected behavior of the release notes script.